entproto/internal/todo/ent/proto/entpb/entpb_grpc.pb.go

```diff
@@ -1,8 +1,4 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
-// versions:
```

Comment:
notice this diff, make sure to use the same env as is used in CI (see readme for instructions), CI will regen all code and make sure no diff exists.